app/views/home/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<section>
+  <h2>
+    Media Spotlight: <%= link_to "#{@sorted_works.first.title}", work_path(@sorted_works.first.id)%> by <%= @sorted_works.first.creator %>
+  </h2>
+  <p><%= @sorted_works.first.votes.length %> votes - <%= @sorted_works.first.description %></p>
```

Comment:
Well done! I like this bit of code. You end up repeating the phrase `@sorted_works.first` a lot here... Maybe it's enough repetition to warrent having a variable dedicated to `@sorted_works.first`